SPECS/pcre/pcre.spec

```diff
@@ -49,15 +53,16 @@ make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
-mv -v %{buildroot}/usr/lib/libpcre.so.* %{buildroot}/lib &&
-ln -sfv ../../lib/$(readlink %{buildroot}/usr/lib/libpcre.so) %{buildroot}/usr/lib/libpcre.so
-ln -sfv $(readlink %{buildroot}/usr/lib/libpcre.so) %{buildroot}/usr/lib/libpcre.so.0
+mv -v %{buildroot}%{_lib}/libpcre.so.* %{buildroot}/lib &&
```

Comment:
I wouldn't replace the `/usr/lib` part of the path with RPM macros. The path is specific for this particular package build, while the macro reflects the file paths on the final system, so that's not exactly the same thing. The package across versions may change the directories it builds. It's also possible in theory (very unlikely), that we will change these macros and this will break a build like this one.